frontend/src/pages/MessagingQueues/MQDetails/MQTables/getTopicThroughputOverview.ts

```diff
@@ -5,11 +5,18 @@ import {
 	MessagingQueueServicePayload,
 	MessagingQueuesPayloadProps,
 } from './getConsumerLagDetails';
+import { TopicThroughputProducerOverviewResponse } from './MQTableUtils';
 
 export const getTopicThroughputOverview = async (
 	props: Omit<MessagingQueueServicePayload, 'variables'>,
 ): Promise<
-	SuccessResponse<MessagingQueuesPayloadProps['payload']> | ErrorResponse
+	| SuccessResponse<
+			(
+				| MessagingQueuesPayloadProps
```

Comment:
this is ideally not a good structure to expect different types of responses based on different conditions.

Reply:
changed